### DISCUSSION TITLE
How to Extract Dataset 1 & 2 from the Given link of data

### DISCUSSION THREAD

**Discussion by AhmedGillani** [christianbirchler-org/sdc-scissor#86] [Q&A] [answered]
Hello Christian Birchler,
             I had a query when I downloaded & extracted the dataset from this link [https://zenodo.org/record/5914130], the data doesn't split in dataset1 and dataset2 as the performance of the ML model shown in the paper are based on dataset 1 & dataset 2 with different RF values. So, I'm a little bit confused about that, how can we extract data?
When I extract the data from the above-mentioned link, some directories pop up i.e., run1, run2, ... like that.
@alessiogambi @ThunderKey @andrius-k @spanichella @skhatiri

**Comment by ChristianBirchler** [comment, marked as answer]
Hi @AhmedGillani, thanks for showing interest in our research; much appreciated!
Regarding your question, we just made Dataset2 publicly available. However, a general issue with BeamNG is that test and code of SDC-Scissor are not backward compatible (BeamNG changes APIs in a non-backward compatible way). This means that we have to regularly update SDC-Scissor to the recent version of BeamNG. In general, if you are interested in running test cases we provide sample test cases that are compatible with SDC-Scissor as well as one of the recent versions of BeamNG.tech. For any inquiry on that issue, I am happy and willing to help you out furthermore.

**Reply by AhmedGillani** [reply]
Thanks, for your quick response. Is it possible, you can share a drive link with me, having csv files of dataset 1 & 2 in it.
So, I can evaluate ML model results from my end and compare it with papers result.

**Reply by ChristianBirchler** [reply]
Unfortunately, the simulator's version has changed since the SANER publication, and those tests can not be run anymore. However, to keep you updated with the recent changes and findings, I highly suggest you refer to the recently accepted manuscript in the "Science of Computer Programming" journal on arXiv: https://doi.org/10.48550/arXiv.2211.11409
